hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotUtils.java

```diff
@@ -242,4 +276,47 @@ public static String getOzonePathKeyForFso(OMMetadataManager metadataManager,
     final long bucketId = metadataManager.getBucketId(volumeName, bucketName);
     return OM_KEY_PREFIX + volumeId + OM_KEY_PREFIX + bucketId + OM_KEY_PREFIX;
   }
+
+  public static SnapshotInfo getLatestGlobalSnapshotInfo(OzoneManager ozoneManager,
```

Comment:
We can not rely on this. This can change immediately after we got this.

Reply:
this is part of the other change https://github.com/apache/ozone/pull/7193 so removed it

Reply:
We don't need this function to be synchronized